fast/stages/2-networking-b-nva/net-landing.tf

```diff
@@ -57,14 +62,31 @@ module "dmz-vpc" {
     subnets_folder = "${var.factories_config.data_dir}/subnets/dmz"
   }
   delete_default_routes_on_create = true
-  routes = {
-    default = {
-      dest_range    = "0.0.0.0/0"
-      next_hop      = "default-internet-gateway"
-      next_hop_type = "gateway"
-      priority      = 1000
-    }
-  }
+  routes = merge(
+    {
+      default = {
+        dest_range    = "0.0.0.0/0"
+        next_hop      = "default-internet-gateway"
+        next_hop_type = "gateway"
+        priority      = 1000
+      }
+    },
+    # Uncomment to enable cross regional VPC traffic
```

Comment:
Is uncommenting enough or does the NVA require some additional programming?
Add a comment in that case, resolve this otherwise.

Reply:
the traffic at the NVA is managed through the default route... no changes are requries at the NVA level to implement cross regiona VPC traffic